pages/dkp/kommander/1.4/release-notes/1.4.1/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.4.1
+title: Release Notes Kommander 1.4.1
+menuWeight: 0
```

Comment:
Do we need to change menuWeight? I recall having to fix some of these so that the ordering of RN is correct